datafusion/expr-common/src/operator.rs

```diff
@@ -142,10 +142,11 @@ impl Operator {
         )
     }
 
-    /// Return true if the operator is a comparison operator.
+    /// Return true if the comparison operator can be used in interval arithmetic and constraint
+    /// propagation
     ///
-    /// For example, 'Binary(a, >, b)' would be a comparison expression.
-    pub fn is_comparison_operator(&self) -> bool {
+    /// For example, 'Binary(a, >, b)' expression supports propagation.
+    pub fn supports_propagation(&self) -> bool {
```

Comment:
this is technically an API change -- Can you please avoid the change and help people upgrade by adding back `is_comparison_operator` that calls `supports_propagation` and mark it deprecated?


https://github.com/apache/datafusion/blob/3353c061830dbd1ef912736fee8426ee0405a3f9/datafusion/core/src/datasource/file_format/parquet.rs#L610-L619